UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManagerTestsScript.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using UnityEngine;
+
+public class UpdateManagerTestsScript : MonoBehaviour
```

Comment:
what does this test do?

Reply:
used for debugging the update manager and removing and adding items

Reply:
measuring performance impact